changelogs/unreleased/config-get-configuration-from-app.md

```diff
@@ -0,0 +1,4 @@
+## feature/config
+
+* Allow to access a configuration from an application script using
+  `config:get()` (gh-8862).
```

Comment:
```suggestion
* It is now possible to access configuration from the application script using the
  `config:get()` function (gh-8862).
```

Reply:
Applied with replacement of `function` to `method`. Thanks!